webapp/commonComponents/expression/expressionEditorPopupState.js

```diff
@@ -121,14 +121,14 @@ export const useAdvancedExpressionEditorPopupState = props => {
 
 export const mapStateToProps = (state, props) => {
```

Comment:
is it possible to get rid of mapStateToProps?

Reply:
sure